wgsl/index.bs

```diff
@@ -1432,7 +1454,11 @@ Note: The only kind of declaration that contain another declaration is a [=funct
 Certain objects are provided by the WebGPU implementation, and are treated as
 if they have been declared before the start of the WGSL program source.
 We say such objects are <dfn noexport>predeclared</dfn>.
-For example, WGSL predeclares [=built-in functions=], and built-in types such as [=i32=] and [=f32=].
+For example, WGSL predeclares:
```

Comment:
Maybe also `attributes`? Of course users can't define these but it might help the reader esp. when the syntax generalizes